NOTICES.md

```diff
@@ -0,0 +1,13 @@
+# Ed-Fi-Standard
+
+This product includes software developed at the [Ed-Fi
+Alliance](https://www.ed-fi.org).
+
+Copyright (c) 2020 Ed-Fi Alliance, LLC and contributors.
+
+This software distribution includes or contains external references to several
+open source packages that are attributed below in this notice. Where required,
+copies of the license agreement are provided in separate files in the Licenses
+subdirectory. Unless noted below, all open source software is distributed in its
+original form without modification.
```

Comment:
For this repository, we can remove the paragraph above as it is not applicable. We need to keep it for other repositories - only this one is special, because it won't actually have any software distribution.